CONTRIBUTING.md

```diff
@@ -83,5 +83,5 @@ You can also:
 
 ```
 cd dsl
-PACT_INTEGRATED_TESTS=1 PACT_BROKER_USERNAME="dXfltyFMgNOFZAxr8io9wJ37iUpY42M" PACT_BROKER_PASSWORD="O5AIZWxelWbLvqMd8PkAVycBJh2Psyg1" PACT_BROKER_HOST="https://test.pact.dius.com.au" go test -run TestPact_Integration
+PACT_INTEGRATED_TESTS=1 PACT_BROKER_USERNAME="dXfltyFMgNOFZAxr8io9wJ37iUpY42M" PACT_BROKER_PASSWORD="O5AIZWxelWbLvqMd8PkAVycBJh2Psyg1" PACT_BROKER_HOST="https://test.pact.dius.com.au" go test -run TestPact_AddInteraction
```

Comment:
Not sure if this is correct either - just the closest test I could find. But then I don't see any uses of the env vars `PACT_INTEGRATED_TESTS`, `PACT_BROKER_USERNAME` or `PACT_BROKER_PASSWORD` in `dsl` dir

These env vars don't seem to be making a diff
```bash
# with env vars
$ PACT_INTEGRATED_TESTS=1 PACT_BROKER_USERNAME="dXfltyFMgNOFZAxr8io9wJ37iUpY42M" PACT_BROKER_PASSWORD="O5AIZWxelWbLvqMd8PkAVycBJh2Psyg1" PACT_BROKER_HOST="https://test.pact.dius.com.au" go test -run TestPact_AddInteraction
PASS
ok      github.com/pact-foundation/pact-go/dsl  10.039s

# without env vars
$ go test -run TestPact_AddInteraction
PASS
ok      github.com/pact-foundation/pact-go/dsl  10.037s
$
```

Reply:
I would just remove steps 1, 2 and 3 from the `Integration Tests` section, they are not longer valid.

Reply:
so the section then would only mention about running `make pact` ?